src/gazebo/ros_gz_description/models/master_ASM/urdf/master_ASM.urdf

```diff
@@ -44,6 +44,74 @@
     </collision>
   </link>
 
+  
+  <joint name="camera_joint" type="fixed">
+        <parent link="base_link"/>
+        <child link="camera_link"/>
+        <origin xyz="0.35 0 0.05" rpy="0 0 0"/>
+    </joint>
+ 
+    <link name="camera_link">
+        <visual>
+            <geometry>
+                <box size="0.03025 0.17525 0.04310"/>
+            </geometry>
+            <!-- <material name="red"/> -->
+        </visual>
+    </link>
+ 
+ 
+    <!-- <joint name="camera_optical_joint" type="fixed">
+        <parent link="camera_link"/>
+        <child link="camera_link_optical"/>

```

Comment:
@pwmarshall Why is there a comment left here?

Reply:
might be needed for tf in future